reviews three/swagger.yaml

```diff
@@ -0,0 +1,954 @@
+      parameters:
+       - $ref: '#/parameters/fields'
+       - $ref: '#/parameters/limit'
+       - $ref: '#/parameters/page'
+       - $ref: '#/parameters/since_id'
+       - $ref: '#/parameters/status'
+       - $ref: '#/parameters/ids'
+      tags:
+        - Get
+      responses:
+        '200':
+           description: Count of draft orders
+           $ref: '#/responses/Integer'
+
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X GET -H "Authorization: Basic YOUR-API-KEY=" "https://your_store_domain.myshopify.com/admin/draft_orders/count.json"
+      
+        - lang: Ruby
+          source: |
+            # load the gem
+            require 'swagger_client'
+            
+            api_instance = SwaggerClient::GetApi.new
+            
+            begin
+              result = api_instance.admin_draft_orders_count_json_get
+              p result
+            rescue SwaggerClient::ApiError => e
+              puts "Exception when calling GetApi->admin_draft_orders_count_json_get: #{e}"
+            end
+
+
+
+  '/admin/draft_orders/#{id}/send_invoice.json':
+    post:
+      summary: Create an invoice for a draft order
+      description: You can use this endpoint to send your customer an invoice via email for thier draft order.  The invoice contains a url link, enabling the customer to pay for the item using web checkout.  Additionally, you can send a `draft_order_invoice` object in the body of the request to override any of the default email parameters for the invoice.  
+      tags:
+        - Create
+      parameters:
+        - name: draft_order_invoice
+          in: body
+          description: draft_order_invoice to post.
+          schema:
+            allOf:
+              - $ref: '#/definitions/draft_order_invoice'
+      
+        - name: id
+          in: path
+          required: true
+          type: integer
+
+
+      responses:
+        '201':
+          description: draft order invoice
+          schema:
+            $ref: '#/definitions/draft_order_invoice'
+            
+            
+      x-code-samples:
+        - lang: curl
+          source: |
+            curl -X POST -H "Content-Type: application/json" -H "Authorization: Basic YOUR-API-KEY=" -d '' "https://your_store_domain.myshopify.com/admin/draft_orders/57760134/send_invoice.json"
+      
+        - lang: Ruby
+          source: |
+            # load the gem
+            require 'swagger_client'
+            
+            api_instance = SwaggerClient::GetApi.new
+            
+            begin
+              result = api_instance.admin_draft_orders_count_json_get
+              p result
+            rescue SwaggerClient::ApiError => e
+              puts "Exception when calling GetApi->admin_draft_orders_count_json_get: #{e}"
+            end      
+            
+            
+            
+  '/admin/draft_orders/#{id}/complete.json':
```

Comment:
Can we indicate that this endpoint isn't available yet somehow? (it will be for launch but not for alpha) a graphic, a picture, something.